<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>change_password</class>
 <widget class="QDialog" name="change_password">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #d9e7ff, stop:1 #e3ffe7);</string>
  </property>
  <widget class="QPushButton" name="logout_password_Btn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>85</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 10px;
	color: white;
	font-size: 50px;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0  rgba(0, 255, 255, 255),, stop:1 #1CB5E0);
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #e3ffe7, stop:1 #d9e7ff);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>â†©</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_password_Btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>430</y>
     <width>85</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 10px;
	color: white;
	font-size: 20px;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0  rgba(0, 255, 255, 255),, stop:1 #1CB5E0);
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #e3ffe7, stop:1 #d9e7ff);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>551</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="Old_pw_label">
        <property name="styleSheet">
         <string notr="true">color: white;
font-size: 20px;
font-weight: 900;
border-radius: 0;
border-width: 0;
background-color: none;
</string>
        </property>
        <property name="text">
         <string>Old password         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="old_pw_line_edit">
        <property name="minimumSize">
         <size>
          <width>289</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white;
font-size: 15px;
color: black;
border-radius: 0;</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="new_pw_label">
        <property name="styleSheet">
         <string notr="true">color: white;
font-size: 20px;
font-weight: 900;
border-radius: 0;
border-width: 0;
background-color: none;</string>
        </property>
        <property name="text">
         <string>New password       </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="new_pw_line_edit">
        <property name="styleSheet">
         <string notr="true">background-color: white;
font-size: 15px;
color: black;
border-radius: 0;</string>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="confirm_pw_label">
        <property name="styleSheet">
         <string notr="true">color: white;
font-size: 20px;
font-weight: 900;
border-radius: 0;
border-width: 0;
background-color: none;</string>
        </property>
        <property name="text">
         <string>Confirm password </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="confirm_pw_line_edit">
        <property name="styleSheet">
         <string notr="true">background-color: white;
font-size: 15px;
color: black;
border-radius: 0;</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
